### PR TITLE
javascript-conditionals - Daniel Hahn

### PR DESCRIPTION
**Pull request by danhahn17**
Completed javascript-conditionals exercise.
Screenshot 2023-03-28 232550

### CODE REVIEW

File: javascript-conditionals/main.js [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+const dan = {
+  name: 'Daniel Hahn',
+  age: 25
+};
+console.log(dan.name + ' ' + 'is old enough to drink and drive:', isOldEnoughToDrinkAndDrive(dan));
+
+function categorizeAcidity(pH) {
+  if (pH === 7) {
+    return 'neutral';
+  } else if ((pH > 0) && (pH < 7)) {
```
Comment (kurtyazdizadeh) [inline javascript-conditionals/main.js]: What happens if `pH` equals `0` or `14`?

**Review by kurtyazdizadeh — CHANGES_REQUESTED** [review]
Let's make sure to take care of all possibilities with the pH function. Great job with logging out some tests/examples!

File: javascript-conditionals/main.js (hunk 1 above)
Comment (danhahn17) [reply javascript-conditionals/main.js]: Oh brain fart - I added the equal to on the wrong else if statement. Correcting it again!

**Review by kurtyazdizadeh — APPROVED** [review]
Approved!

**Pull request merged**